packages/core/src/pages/index.tsx

```diff
@@ -87,9 +87,7 @@ export const getStaticProps: GetStaticProps<
 > = async ({ previewData }) => {
   const [page, globalSections] = await Promise.all([
     getPage<PageContentType>({
-      ...(previewData?.contentType === 'home'
-        ? previewData
-        : { filters: { 'settings.seo.slug': '/' } }),
+      ...(previewData?.contentType === 'home' && previewData),
```

Comment:
Why have you removed the fallback filters?

Reply:
But shouldn't it be the case to correct the filter and not remove it?

Reply:
I see, makes sense. Maybe this was legacy code, before we had a contentType por home specifically and had to use the slug to determine which landing page was the home.